lib/types.ts

```diff
@@ -191,6 +191,7 @@ export interface TaskSys {
 
 export interface TaskInputData {
   assignedToId: string
+  assignedToType?: 'User' | 'Team'
```

Comment:
why does this take an `assignedToId` that is a `string` as a value here? :/ I'd expect this to be consistent with the public interface of the API and have instead an `assignedTo: Link<User | Team>`

Reply:
i guess it'd be a breaking change so this is probably the best we can do

Reply:
@FLoppix do we want to do lowercase or uppercase for `assignedToType`?